app/assets/javascripts/hyrax/channels/notifications.js

```diff
@@ -0,0 +1,27 @@
+App.cable.subscriptions.create("Hyrax::NotificationsChannel", {
+    connected: function(data) {
+        this.perform("update_locale", { locale: $('html').attr('lang') })
```

Comment:
This persists the current locale value the user has chosen to the User model, which the notifications can then use when rendering a new notification. This is needed because the code that processes notifications does not have access to session -- indeed, a user may not even be connected when a notification fires. This also moves us in the direction of being able to allow users to set their own preferred locale and have it 'stick'.